### PR TITLE
Reword the minting section to make clear it's implementation dependent

### PR DESCRIPTION
**Pull request by arnau**
Reword the minting section to make clear it's implementation dependent.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -988,16 +988,11 @@ To verify the consistency of two versions of a register, given that total-entrie
...
-  To mint a new entry in the register:
-  * POST a new item in [[#json-representation]] to the ..mint url.. with the proposed new entry number.
-  * The item will first appear as a new item in the store
-  * <em> .. status codes and error cases </em>
-  * <em> .. 202 Accepted with a Location header of the [[#entry-resource]] for the new entry</em>
-  * The proposer can only trust it has been secured when the entry
-    is listed in the [[#entries-resource]] with the same [[#entry-number-field]],
-    and the [[#entry-resource]] is a covered by one or more [[#digital-proofs]] for the register.
-
-The item MUST NOT contain empty fields.
```
Comment (MatMoore) [inline index.bs:970]: Is this part an implementation detail of minting or does it say something about what you can store in a register? Assuming the former but just want to check.

File: index.bs [hunk 2]
```diff
@@ -988,16 +988,11 @@ To verify the consistency of two versions of a register, given that total-entrie
...
+Minting refers to the notion of creating a new entry, stamping it with a
+timestamp and appending it to the log.
+
+This specification doesn't define any way to do that as it is seen as an
+implementation concern.
```
Comment (MatMoore) [inline index.bs:965]: I think this could be worded more strongly, like "it is up to the implementation how to do this"

**Review by MatMoore — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (arnau) [reply index.bs:970]: All that should be derived from the requirements for the different interfaces (i.e. entry resource, digital proofs resources).

File: index.bs (hunk 2 above)
Comment (arnau) [reply index.bs:965]: @bahmady what do you think?

Comment (arnau) [reply index.bs:965]: To clarify, I agree that the wording should be stronger, I'm not entirely sure the style aligns with the spec.

**Review by michaelabenyohai — APPROVED** [review]

**Pull request merged**